device-types/Nokia/7210-SAS-T-12F10T-4XFP.yaml

```diff
@@ -0,0 +1,71 @@
+---
+manufacturer: Nokia
+model: 7210 SAS-T 12F10T 4XFP
+slug: nokia-7210-sas-t-12f10t-4xfp
+part_number: 3HE08116AAAB01
+u_height: 2
+is_full_depth: false
```

Comment:
Please add the `weight` and `weight_unit` attribute if you have it available

Reply:
Added